app/services/login/after_login_actions.rb

```diff
@@ -21,6 +21,10 @@ def perform
       update_account_login_stats(login_type)
       id_mismatch_validations
 
+      if Flipper.enabled?(:reactivation_experiment_initial_gate, current_user)
```

Comment:
Could probably tighten this up as:

`AcceptableVerifiedCredentialAdoptionService.new(@current_user).perform if Flipper.enabled?(:reactivation_experiment_initial_gate, current_user)`

Reply:
Changed!